add-on/manifest.firefox.json

```diff
@@ -1,18 +1,20 @@
 {
-  "browser_action": {
+  "action": {
     "browser_style": false
   },
   "options_ui": {
     "browser_style": false
   },
-  "applications": {
+  "background": {
+    "scripts": ["dist/bundles/backgroundPage.firefox.bundle.js"]
+  },
+  "browser_specific_settings": {
     "gecko": {
       "id": "ipfs-firefox-addon@lidel.org",
-      "strict_min_version": "91.1.0"
+      "strict_min_version": "111.0.0"
```

Comment:
:thought_balloon: @whizzzkid   is this a bug or a coincidence that both Chrome and Firefox use v111 here?
iirc Firefox shipped MV3 in v109, are we depending on some fixes that landed later?